server/pages/index.js

```diff
@@ -528,6 +528,47 @@ module.exports = function() {
 		} );
 	} );
 
+	app.get( '/support-user', function( req, res ) {
+		// Do not iframe
+		res.set( {
+			'X-Frame-Options': 'DENY',
+		} );
+
+		if ( ! req.cookies.wordpress_logged_in ) {
+			// Not logged in, 404!
+			return render404( req, res );
```

Comment:
@gwwar Transferring over my [comment from original PR](https://github.com/Automattic/wp-calypso/pull/21674#discussion_r164684068) here. I don't think we can `render404` here because of 3rd party resources that are (or might be in the future) loaded with it. It seems like we should only render `support-user.pug` here, and maybe some other clean custom template that we add, but not the existing ones. Anyway, let me know what do you think about this?

Reply:
ah true, let's maybe add a stripped down 404 page :)

Reply:
Another option might be to `res.redirect( '/support-user' )` here to strip query params, and render the normal 404